ports/esp32/machine_pwm.c

```diff
@@ -71,28 +71,31 @@ STATIC ledc_timer_config_t timers[PWM_TIMER_MAX];
 #define TIMER_IDX_TO_MODE(timer_idx) (timer_idx / LEDC_TIMER_MAX)
 #define TIMER_IDX_TO_TIMER(timer_idx) (timer_idx % LEDC_TIMER_MAX)
 
-// Params for PWM operation
+// Params for PW operation
```

Comment:
I think having "PWM" is better, because "PW" is not a common abbreviation and may confuse readers.

Reply:
That is something, that was fixed in the previous PR and now just returns, because my PR is based on an earlier state of the code.